tests/tests_default.yml

```diff
@@ -2,6 +2,38 @@
 ---
 - name: Ensure that the role runs with default parameters
   hosts: all
-  gather_facts: false  # test that role works in this case
+  become: true
   roles:
-    - linux-system-roles.template
+    - linux-system-roles.journald
+
+  tasks:
+    - name: Get systemd-journald service state
+      command:
+        cmd: systemctl show --property SubState systemd-journald.service
+      register: systemctl
+      changed_when: false
+
+    - name: Verify that systemd-journald is running
+      assert:
+        that: systemctl.stdout == 'SubState=running'
+
+    - name: Verify that /run/log/journal/$MACHINE_ID/system.journal exists
+      command:
```

Comment:
better to use the `stat` module here - https://docs.ansible.com/ansible/latest/collections/ansible/builtin/stat_module.html#ansible-collections-ansible-builtin-stat-module
```yaml
    - name: Verify that /run/log/journal/$MACHINE_ID/system.journal exists
      stat:
        path: /run/log/journal/{{ ansible_machine_id }}/system.journal
      register: __stat
      failed_when: not __stat.stat.exists
```